pyproject.toml

```
@@ -1,13 +1,20 @@
 [project]
 name = "m-gsm-symbolic"
-version = "0.1.0"
-description = "Add your description here"
+version = "0.0.1"
+description = "Evaluating cross-lingual reasoning transfert"
 readme = "README.md"
 authors = [
     { name = "Kenneth Enevoldsen", email = "kennethcenevoldsen@gmail.com" },
+    { name = "Nicolas Legrand", email = "nicolas.legrand@cas.au.dk" },
+]
+requires-python = ">=3.10,<3.13"
+dependencies = [
+    "ipykernel>=6.29.5",
+    "torch",
+    "unsloth",
```

let us add a lower bound here
```suggestion
    "torch>=2.6.0",
    "unsloth>=2025.5.2",
```